app/src/main/java/com/example/caculator/Caculator.kt

```diff
@@ -0,0 +1,18 @@
+package com.example.caculator
+
+class Calculator {
```

Comment:
change class -> object because it only has one helper function
https://kotlinlang.org/docs/tutorials/kotlin-for-py/objects-and-companion-objects.html